pkg/api/convertertoapi.go

```diff
@@ -683,6 +683,7 @@ func convertVLabsKubernetesConfig(vlabs *vlabs.KubernetesConfig, api *Kubernetes
 	api.DNSServiceIP = vlabs.DNSServiceIP
 	api.ServiceCIDR = vlabs.ServiceCidr
 	api.NetworkPlugin = vlabs.NetworkPlugin
+	api.NetworkMode = vlabs.NetworkMode //TODO: why no networkPolicy?
```

Comment:
See the `setVlabsKubernetesDefaults` func that deals w/ back-compat validation.

In fact given the order of func invocations in `convertVLabsOrchestratorProfile` adding a simple vlabs NetworkPolicy --> unversioned NetworkPolicy assignment wouldn't break anything, as the `setVlabsKubernetesDefaults` will overwrite them.

I think the actual // TODO is to remove the `api.NetworkPlugin = vlabs.NetworkPlugin` line here, as that assignment is happening in `setVlabsKubernetesDefaults` in the "no back compat translation required" condition.

@CecileRobertMichon could you do a 5 minute spot check on my conclusions above?

Reply:
thanks Jack. Will remove the comments

Reply:
Sorry I didn't see this earlier... Yeah I think `networkPolicy` is special because it used to be called `networkPlugin` so we had to do some dark magic to maintain back compat 

I vote we don't touch it, it's working as-is and I don't think an extra line of code is worth the risk.